app/enterprise/2.1.x/developer-portal/administration/application-registration/azure-oidc-config.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Set up External Portal Application Authentication with Azure AD and OIDC
+---
+
+## Overview
+
+These instructions help you set up Azure AD as your third-party identity provider
+for use with the Kong OIDC and Portal Application Registration plugins.
+
+### Prerequisites
+
+- The `portal_app_auth` configuration option is configured for your OAuth provider
+  and strategy (`kong-oauth2` or `external-oauth2`). See
+  [Configure the Authorization Provider Strategy](/enterprise/{{page.kong_version}}/developer-portal/administration/application-registration/#portal-app-auth) for the Portal Application Registration plugin.
+
+## Create an Application in Azure
+
+1. Within Azure, go to the **App Registrations** service and register a new application.
+
+   ![Azure App Registrations](/assets/images/docs/dev-portal/ms-azure-app-reg.png)
+
+1. In **Certificates & secrets**, create a Client secret and save it in a
+   secure location. You can only view the secret once.
+
+1. Under **Manifest**, update `accessTokenAcceptedVersion=2` (default is null).
+   The JSON for your application should look similar to this example:
+
+  ![Azure Manifest](/assets/images/docs/dev-portal/azure-manifest.png)
+
+## Create a Service and a Route in Kong
+
+1. Create a Service example:
+
+   ```bash
+   http put :8001/services/httpbin-service-azure url=https://httpbin.org/anything
```

Comment:
```suggestion
   $ http PUT :8001/services/httpbin-service-azure \
      url=https://httpbin.org/anything
```
Capitalize the method for consistency.

You can format any bash command example with `\` to denote arbitrary line breaks for readability. They won't change anything in the command itself, as long as you break in the middle of a name or a URL, for example.